tests/Rector/ClassMethod/DataProviderAnnotationToAttributeRector/Fixture/external-provider-fqcn.php.inc

```diff
@@ -0,0 +1,47 @@
+<?php
```

Comment:
This test is the same like below, isn't it?

Reply:
There is just a single backslash more :-)

Reply:
See https://github.com/rectorphp/rector-phpunit/pull/167#discussion_r1167766572